Sep  4 21:32:51 webapp-10 simplesamlphp-idp[16242]: 3 [e69bb4b4e7] SimpleSAML\Error\Exception: Error 8 - Undefined index: name at /apps/installation/simplesamlphp/simplesamlphp-master/modules/statistics/lib/Statistics/FieldPresentation/Entity.php:20

Sep  4 21:32:51 webapp-10 simplesamlphp-idp[16242]: 3 [e69bb4b4e7] Caused by: TypeError: Argument 1 passed to SimpleSAML\XHTML\Template::getEntityDisplayName() must be of the type array, null given, called in /apps/installation/simplesamlphp/simplesamlphp-master/modules/statistics/lib/Statistics/FieldPresentation/Entity.php on line 20

--- a/lib/Statistics/FieldPresentation/Entity.php
+++ b/lib/Statistics/FieldPresentation/Entity.php
@@ -17,7 +17,7 @@ class Entity extends Base
         $translation = ['_' => 'All services'];
         foreach ($this->fields as $field) {
             if (array_key_exists($field, $metadata)) {
-                $translation[$field] = $this->template->getEntityDisplayName($metadata[$field]['name']);
+                $translation[$field] = $this->template->getEntityDisplayName($metadata[$field]);
             }
         }
         return $translation;

